yarn-project/cli-wallet/src/utils/wallet.ts

```diff
@@ -208,4 +223,23 @@ export class CLIWallet extends BaseWallet {
     printGasEstimates(fee, limits, this.userLog);
     return simulationResults;
   }
+
+  // Recently added when having the wallet instantiate PXE as PXE is now hidden. This forced me to expose this when
+  // refactoring `checkTx` cli-wallet command.
+  getContracts(): Promise<AztecAddress[]> {
+    return this.pxe.getContracts();
+  }
+
+  // Recently added when having the wallet instantiate PXE as PXE is now hidden. This forced me to expose this when
+  // refactoring `checkTx` cli-wallet command.
+  getNotes(filter: NotesFilter): Promise<UniqueNote[]> {
+    return this.pxe.getNotes(filter);
+  }
+
+  // Recently added when having the wallet instantiate PXE as PXE is now hidden. This forced me to expose this when
+  // refactoring `bridge_fee_juice` cli-wallet command.
+  // TODO: This should most likely just be refactored to getProtocolContractAddresses.
+  getPXEInfo(): Promise<PXEInfo> {
+    return this.pxe.getPXEInfo();
+  }
```

Comment:
WDYT about these commands? Needed to expose them as we had the corresponding commands exposed on the CLI (getNotes was required by inspectTx and it prints out notes emitted in the tx).

As a dev being able to print out contracts and notes when debugging seems pretty useful.